### DISCUSSION TITLE
EventId property cannot be used

### DISCUSSION THREAD

**Discussion by bobskigit** [ravendb/ravendb#14153] [Q&A]
RavenDB seems not to like an object called EventId. We use ravenDB for logging and it seems impossible to create an index  with EventId.Id as one of the properties.

  var query = _dbSession.Query<RavenLogEntry>()                        
                .Where(le => le.TimeStamp >= startTime &&
                             le.TimeStamp <= endTime &&
                             le.EventId.Id.In(eventIds))
                .Select(le => new LogEntrySummary
                {
                    Id = le.Id,
                    Text = le.Text,
                    TimeStamp = le.TimeStamp,
                    Level = le.Level,
                    Category = le.Category,
                    //EventId = le.EventId.Id,    // Causes **exception**
                    Event_Id = le.EventId.Id,
                    //DebugEventId = le.EventId,                            
                });

**Comment by mcrio** [comment]
Hi, I guess it'd help if you provide a simple failing test case.

**Reply by garayx** [reply]
Also please share the full exception (+stacktrace) that you are getting
